osu.Game/Screens/Select/Carousel/DrawableCarouselItem.cs

```diff
@@ -105,7 +103,7 @@ protected override void Update()
 
         protected virtual void UpdateItem()
         {
-            if (item == null)
+            if (item == null || Item == null)
```

Comment:
Really this should be

```suggestion
            if (Item == null)
```

Checking both is just redundant.